pom.xml

```diff
@@ -264,6 +264,12 @@
     <version.org.mvel>2.4.3.Final</version.org.mvel>
     
     <version.ch.obermuhlner>2.0.1</version.ch.obermuhlner>
+    <version.javax.xml.bind>2.3.1</version.javax.xml.bind>
```

Comment:
javax.xml.bind can be removed as it was already removed - and it is not anymore in kie-parent